app.py

```diff
@@ -58,34 +137,66 @@ def index():
 # will have to be updated
 @app.route("/registration", methods=["GET", "POST"])
 def registration():
-    """
-    The registration page of the app. Allows new users to
-    register for an account.
-    """
```

Comment:
The function docstrings here and in `login` were intended both as a short documentation of the functions, and to suppress a linter warning. I'm unsure if they were removed or if you were working with an outdated version of main, but including these should suppress the linter warning.

Reply:
Thank you!